src/services/auth/auth-service.ts

```diff
@@ -0,0 +1,190 @@
+import { makeGetApiCall, makePostApiCall, makePutApiCall } from '../request/request-service';
+import { createFullApiUrl } from '../utils/utils-service';
+import _ from 'lodash';
```

Comment:
Let's put the import of lodash in the utils file, so that if we wanna change it in the future, we can change it in one place